pkg/scaffold/scaffold.go

```diff
@@ -181,7 +181,7 @@ func (s *Scaffold) doRender(i input.Input, e input.File, absPath string) error {
 		}
 	}
 	_, err = f.Write(b)
-	log.Infoln("Created", i.Path)
+	log.Infof("Created '%s' (%s)\n", i.Path, absPath)
```

Comment:
Undo this change (outside of the scope of this PR).

Reply:
sure, let me revert it

Reply:
done in 1c31dac